### PR TITLE
Build content for Defense Evasion-Rootkit

### PR DESCRIPTION
**Pull request by hemantvish97**
Created new usecase for Skywiper or Flame malware

### CODE REVIEW

File: workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    - How was the Skywiper or Flame file injected into the system?
+
+    - Is there any system or user has compromised?
+
+    ### Detection and Analysis
+
+    - Examine the Sysmon logs for this malware string "mssecmgr.ocx" and user details, and if a suspicious file is observed, check its hash value on Virustotal. This can help identify any suspicious activity and provide valuable information for further analysis.
+
+    - To detect Skywiper or Flame malware injection, check Sysmon process logs for activities like process added into th system.
```
Comment (aman1808) [inline workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml:35]: *into the system. Avoid spelling mistakes

File: workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    - Check the last 24 hours logs and identify the behaviour of user and system via monitoring the systemand check any unsual IAM activity has been performed or any known user behaviour is suspicious.
```
Comment (aman1808) [inline workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml:37]: the system and check any unusual. Avoid spelling mistakes

File: workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    - Check the last 24 hours logs and identify the behaviour of user and system via monitoring the systemand check any unsual IAM activity has been performed or any known user behaviour is suspicious. 
+
+    ### Contain, Eradicate and Recover
+
+    - If the user is unknow and the file is malicious then qurantine that file and raise this to security team.
+
+    - If malware injection is detected and the process has aaded then stop the suspicious process and then isolate the affected system(s) from the network to prevent further damage by disabling network access, It is important to escalate the incident to senior security engineers for further investigation and response.
+
+    - If any unknow activity observed than take action on it and then raise this to admin team.
```
Comment (aman1808) [inline workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml:45]: *unknown

File: workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml (hunk 1 above)
Comment (hemantvish97) [reply workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml]: Done with the changes.

File: workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml (hunk 2 above)
Comment (hemantvish97) [reply workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml]: Done with the changes.

File: workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml (hunk 3 above)
Comment (hemantvish97) [reply workbooks/Advanced Threat Detection/Windows Process Monitoring/skywiper_or_flame_file_injected_3f30dc2dd2a945b084c6ceb9bc47122e.yml]: Done with the changes

**Review by aman1808 — APPROVED** [review]

**Pull request merged**